chapter_linear-networks/fashion-mnist_vn.md

```diff
@@ -52,7 +58,8 @@ FashionMNIST consists of images from 10 categories, each represented by 6k image
 Consequently the training set and the test set contain 60k and 10k images, respectively.
 -->
 
-*dịch đoạn phía trên*
+FashionMNIST chứa các hình ảnh từ 10 danh mục, mỗi danh mục được thể hiện bằng sáu nghìn tấm ảnh trong tập huấn luyện và một nghìn tấm ảnh trong tập kiểm tra. 
```

Comment:
```suggestion
FashionMNIST chứa các hình ảnh từ 10 lớp, mỗi danh mục được thể hiện bằng sáu nghìn tấm ảnh trong tập huấn luyện và một nghìn tấm ảnh trong tập kiểm tra. 
```

Trong bài toán đa lớp thì nhóm thống nhất dịch `category` là `lớp` để đỡ bối rối khi đọc